### PR TITLE
FT: Allow ring policies

### PR DESCRIPTION
**Pull request by alexandre-merle**
Allowing ring policies

### CODE REVIEW

**Comment by alexandre-merle** [comment]
@ironman-machine try

File: lib/auth/v4/validateInputs.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ function validateCredentials(credentials, timestamp, log) {
            { scopeDate, timestampDate });
         return errors.RequestTimeTooSkewed;
     }
-    if (service !== 's3' && service !== 'iam') {
+    if (service !== 's3' && service !== 'iam' && service !== 'ring') {
         log.warn('service in credentials is not s3', { service });
```
Comment (rahulreddy) [inline lib/auth/v4/validateInputs.js]: You can amend this log line as well to `service in credentials is not one of s3/iam/ring`

**Comment by alexandre-merle** [comment]
@rahulreddy updated :)

**Comment by alexandre-merle** [comment]
Changed target branch to rel/6.4

**Comment by rahulreddy** [comment]
End to end http://ci.ironmann.io/gh/scality/Integration/9585

**Comment by DavidPineauScality** [comment]
Merging on behalf of the maintainer

**Pull request merged**